homeassistant/components/media_player/plex.py

```diff
@@ -272,93 +527,383 @@ def _active_media_plexapi_type(self):
     @property
     def media_content_id(self):
         """Content ID of current playing media."""
-        if self.session is not None:
+        if self.session:
             return self._convert_na_to_none(self.session.ratingKey)
 
     @property
     def media_content_type(self):
         """Content type of current playing media."""
-        if self.session is None:
+        if self.state not in (STATE_IDLE, STATE_OFF):  # true or frozen state
+            if self.session:
+                media_type = self.session.type
+                if media_type in ('episode', 'clip'):
+                    if media_type == 'clip':
+                        _LOGGER.debug('Clip content type detected, '
+                                      'compatibility may vary: %s',
+                                      self.entity_id)
+                    return MEDIA_TYPE_TVSHOW
+                elif media_type == 'movie':
+                    return MEDIA_TYPE_VIDEO
+                elif media_type == 'track':
+                    return MEDIA_TYPE_MUSIC
             return None
-        media_type = self.session.type
-        if media_type == 'episode':
-            return MEDIA_TYPE_TVSHOW
-        elif media_type == 'movie':
-            return MEDIA_TYPE_VIDEO
-        elif media_type == 'track':
-            return MEDIA_TYPE_MUSIC
-        return None
+
+    @property
+    def media_artist(self):
+        """Artist of current playing media, music track only."""
+        if self.state not in (STATE_IDLE, STATE_OFF):  # true or frozen state
+            if self.media_content_type is MEDIA_TYPE_MUSIC:
```

Comment:
Are you referring to the "is MEDIA_TYPE MUSIC"?  If so, can you explain when I should use is vs equals?